### PR TITLE
Azure CI: Remove empty MklDnnFunctionalTests

### PR DESCRIPTION
**Pull request by azhogov**

### CODE REVIEW

File: .ci/azure/linux.yml [hunk 1]
```diff
@@ -125,6 +125,7 @@ jobs:
     displayName: 'nGraph UT'
     continueOnError: false
 
+    # python3 $(WORK_DIR)/gtest-parallel/gtest_parallel.py $(BIN_DIR)/InferenceEngineUnitTests --workers=$(WORKERS_NUMBER) --dump_json_test_results=InferenceEngineUnitTests.json --gtest_filter=*smoke* -- --gtest_print_time=1
```
Comment (SDxKeeper) [inline .ci/azure/linux.yml:128]: why this change in removing mkl dnn functional tests?

Comment (azhogov) [reply .ci/azure/linux.yml:128]: it is empty

**Comment by azhogov** [comment]
> Please, perform the same for `ClDnnFunctionalTests`

Removed internally
